canvas.go

```diff
@@ -249,8 +233,10 @@ c73.985,0,73.985,0,73.985-73.986V72.986C84.979-1,84.979-1,10.994-1z`,
 	for _, relation := range c.relations {
 		relation.definition(canvas)
 	}
+	c.iconsRendered = make(map[string]bool)
+	c.iconIds = make(map[string]string)
```

Comment:
I went with the former because they are needed outside the scope of that method.  However, they do work much better away from there.  Thanks.